api/src/Page/DC.php

```diff
@@ -986,8 +990,8 @@ function _data_collections($single = null)
 
                 if ($dc['DCT'] == 'Mesh')
                     $dc['DCT'] = 'Grid Scan';
-                if ($dc['DCT'] == 'OSC')
-                    $dc['DCT'] = 'Data Collection';
+                if ($dc['DCT'] == 'OSC' || $dc['DCT'] == 'Diamond Anvil High Pressure') 
+                    $dc['DCT'] = 'Single Crystal';
```

Comment:
Because you are no longer setting OSC to Data Collection I think you need to add Single Crystal to dc.html line 29. So it works for non xpdf.

Also how would you feel about not doing this change to single crystal but passing either OSC or Diamond Anvil High Pressure to the front end then changing `collectionTypeSpecific: { "Single Crystal": MXView },` to `collectionTypeSpecific: { "Diamond Anvil High Pressure": MXView ,
                                         "OSC": MXView },

Reply:
That makes sense, it's a good bit of futureproofing